app/controllers/data_requests_controller.rb

```diff
@@ -1,4 +1,5 @@
 class DataRequestsController < ApplicationController
+  before_action :verify_scope
```

Comment:
Minor: the scope terminology is confusing, `verify_scope` uses `unscoped_exportable` to determine the 404 response. Perhaps `verify_exportable_resource` ?

Reply:
that's a clearer alternative